src/getWeatherData.js

```diff
@@ -21,10 +18,10 @@ const getWeatherData = async (info) => {
     //   current time
     const localTimeOffset = new Date().getTimezoneOffset() * 60;
     const currentDate = new Date(
-      new Date() - (-localTimeOffset - weatherData.timezone) * 1000
+      new Date() - (-localTimeOffset - weatherData.timezone) * 1000,
     );
     const theTime = currentDate.toLocaleTimeString('en-GB').slice(0, -3);
-    gatheredData.current_time = theTime;
+    gatheredData.currentTime = theTime;
     //   main condition
     gatheredData.icon = weatherData.weather[0].icon;
     gatheredData.description = weatherData.weather[0].description;
```

Comment:
You can make your code more organized taking advantage of `Object.assign()` here. Example:
`Object.assign( gatheredData, {
city: weatherData.name,
currentTime: theTime,
icon: weatherData.weather[0].icon,
... 
})`